CHANGELOG.md

```diff
@@ -1,3 +1,23 @@
+# [v0.4.0]
+
+## Fixed
+
+- Fixed potential overflow in calls to `align_up`.
```

Comment:
I think we have an issue for this one as well: https://github.com/rust-vmm/linux-loader/issues/66.